xml/depl_maintenance.xml

```diff
@@ -1503,6 +1503,295 @@ migrate</screen>
     the selected nodes.
    </para>
   </sect2>
+  <sect2 xml:id="sec.depl.maintenance.upgrade.troubleshooting">
+   <title>Common Upgrade Problems and Possible Solutions</title>
+    <qandaset>
+     <qandaentry>
+      <question>
+       <para>
+	Upgrade of the admin server has failed.
+       </para>
+      </question>
+      <answer>
+       <para>
+	Check for empty, broken, and not signed repositories in the &admserv;
```

Comment:
"or" instead of "and"?

Reply:
I think _and_ is better, as you should check for all these symptoms and not just for one or two. Am I wrong?

Reply:
Well, right. Your call. I guess my feeling for English is probably too much influenced by my Czech thinking :-)